src/Mesh/babylon.groundMesh.ts

```diff
@@ -5,6 +5,8 @@
         private _worldInverse = new Matrix();
         private _heightQuads: { slope: Vector2; facet1: Vector4; facet2: Vector4 }[];
         public _subdivisions: number;

```

Comment:
The public getter can stay, the private can be removed. The public will anyhow return a wrong value, if x != y. Return math.min(x,y), this will be the safest.